providers/system_proxy.rb

```diff
@@ -253,10 +251,6 @@ def remove_trailing_slash(uri)
           node.normal['job_status'][jid]['status'] = 0
         end
 
-        gecos_ws_mgmt_jobids 'network_res' do
```

Comment:
Could you confirm this piece of code is not necessary anymore?

Reply:
Yes. This is a bugfix.

This piece of code is in the "Nothing to do!" section. This was clearly a bug. When the system_proxy settings does not change, this piece of code restarts the network manager. So every time the Chef client was executed the network manager was restarted.